SIG.md

```diff
@@ -0,0 +1,85 @@
+
+SIGs don't have any additional authority - they are expected to drive towards solutions that can be accepted by the project's committer community - ideally by consensus, and if necessary, a vote. 
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+
+## SIG Lead
+
+- Membership tracked in [sigs.yaml](./sigs.yaml)
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST organize a method of keeping cadence (meetings, async channel in Cilium Slack, etc.), and make sure [sigs.yaml](./sigs.yaml) is up to date.
+- SHOULD facilitate cadence but MAY delegate to other members or future Leads/Leads in training.
+- MUST ensure notes made available.
+- MAY delegate the actual communication and creation of content to other contributors where appropriate.
+
+### Requirements
+Leads MUST be at least a "reviewer" on our [contributor ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer) to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
+
+### On-boarding and Off-boarding Leads
+Leads MAY decide to step down at anytime and propose a replacement. Use lazy consensus amongst the other Leads with fallback on majority vote to accept the proposal. The candidate SHOULD be supported by a majority of SIG contributors.
+Leads MAY select additional Leads through a super-majority vote amongst Leads. This SHOULD be supported by a majority of SIG contributors.
+
+## Other Roles
+This governance document outlines the required roles for SIGs; however, SIGs are allowed to operate how they see fit outside of minimum governance requirements, including defining more roles to sustain the group. If a SIG needs to change the Lead position to include or remove duties, this needs to be approved by the committers. Newly created roles that don't assume any responsibility of Leads should follow the governing processes in the SIG's charter.
+
+# SIG Creation
+
+## Goals
+The primary goal of the charters is to define the scope of the SIG within Cilium and how the SIG leaders exercise ownership of these areas by taking care of their responsibilities. A majority of the effort should be spent on these concerns. SIG Leads are also reponsible for driving reviews for CFPs for their focus area, and have a responsibility to bring concerns to committers if they are unsure. 
```

Comment:
What does "driving reviews" mean?

Currently we primarily rely upon GitHub automated review assignment for PR review (at least for the most popular repositories). While this isn't a perfect system, I think it can help with distributing the workload.

Probably as a fallback it makes sense for the Lead to keep an eye on the reviews, but in a well-functioning SIG I would hope that we are able to distribute the load between the folks in the team to ensure we don't overload the Leads.

(This may be something we end up just settling on some wording for, then work with the initial SIGs to clarify / document expectations).

Reply:
fair, i think driving review overall means that a CFP will be responded to in a timely manner, something hands off for the leads ultimately (gentle pokes to whoever is reviewing within their SIG), making sure conversation doesn't go stale? I do realize that adds a lot of glue work for the leads though, and i dont want the work to grow unmeasured.